bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/DialogProcessor.java

```diff
@@ -362,12 +362,14 @@ public synchronized void sttEventReceived(STTEvent sttEvent) {
             logger.debug("RecognitionStopEvent event received");
             toggleProcessing(false);
         } else if (sttEvent instanceof SpeechRecognitionErrorEvent sre) {
-            logger.debug("SpeechRecognitionErrorEvent event received");
+            String message = sre.getMessage();
+            logger.debug("SpeechRecognitionErrorEvent event received: {}", message);
             if (!isSTTServerAborting) {
                 abortSTT();
                 toggleProcessing(false);
-                String text = i18nProvider.getText(bundle, "error.stt-error", null, dialogContext.locale());
-                say(text == null ? sre.getMessage() : text.replace("{0}", sre.getMessage()));
+                if (!message.isEmpty()) {
+                    say(message);
+                }
```

Comment:
In that case, the message "error.stt-error" is never used and can be removed.

Please also check that all existing STT services are appropriate with this change.